engines/experiment/plugin/runner/rpc.go

```diff
@@ -0,0 +1,62 @@
+package runner
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/gojek/turing/engines/experiment/plugin/shared"
+	"github.com/gojek/turing/engines/experiment/runner"
+)
+
+// rpcClient implements ConfigurableExperimentRunner interface
+type rpcClient struct {
+	shared.RPCClient
+}
+
+func (c *rpcClient) Configure(cfg json.RawMessage) error {
+	return c.Call("Plugin.Configure", cfg, new(interface{}))
+}
+
+func (c *rpcClient) GetTreatmentForRequest(
+	ctx context.Context,
+	log runner.Logger,
+	header http.Header,
+	payload []byte,
+) (*runner.Treatment, error) {
+	req := GetTreatmentRequest{
+		Context: ctx,
```

Comment:
You were saying earlier that there were issues serializing the logger and Go context. Is this working now?

Reply:
Changes to the runner interface will be introduced in a separate PR. I didn't want to mix it in into a somehow big PR. 

>  Is this working now?
It works for imported experiment engines (compile-time plugins), but doesn't work for the RPC plugins.